voice/trigger-notify-action/index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+use \Psr\Http\Message\ServerRequestInterface as Request;
+use \Psr\Http\Message\ResponseInterface as Response;
+
+require 'vendor/autoload.php';
+
+$app = new \Slim\App;
+$app->get('/webhooks/answer', function (Request $request, Response $response) {
+    $params = $request->getQueryParams();
+    $uri = $request->getUri();
+    $ncco = [
+        [
+            'action' => 'talk',
+            'text' => 'Thanks for calling the notification line'
+        ],
+        [
+            'action' => 'notify',
+            'payload' => ['foo' => 'bar'],
+            'eventUrl' => [
+                $uri->getScheme().'://'.$uri->getHost().':'.$uri->getPort().'/webhooks/dtmf'
```

Comment:
Should this URL be `/webhooks/notification` ? I don't see `/webhooks/dtmf` defined.